e2e/davinci-app/tsconfig.app.json

```diff
@@ -4,6 +4,6 @@
     "outDir": "../../dist/out-tsc",
     "moduleResolution": "Bundler"
   },
-  "exclude": ["src/**/*.spec.ts", "src/**/*.test.ts"],
-  "include": ["src/**/*.ts"]
+  "exclude": ["**/*.spec.ts", "**/*.test.ts"],
+  "include": ["./main.ts", "components/**/*.ts"]
```

Comment:
was causing issues with module resolution not using Bundler since there is no `src` dir here